### ISSUE TITLE
[XamarinProfiler-Windows]: Accessibility: Color Contrast: MAS1.4.3: "Xamarin Profiler" window name is not having color contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#424]
### Pre-Requisite:
"1. Install Xamarin Profiler.
2. Launch Profiler through ".mlpd" file.
### Steps to Reproduce

1. Follow the Pre-requisite to launch the xamarin profiler.
2. Verify the color contrast ratio of "Xamarin Profiler" text i.e, Window name.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 4.0:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screenshot
![1 launch page](https://user-images.githubusercontent.com/22956130/41703759-0dbfd920-7552-11e8-850f-18e416e19855.png)

### Environment
Microsoft Windows version: 1803(OS Build 17134.112)
Xamarin Profiler version: 1.6.3.342

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-Windows
Wipro
